tidb.go

```diff
@@ -217,7 +217,7 @@ func GetRows4Test(ctx context.Context, sctx sessionctx.Context, rs ast.RecordSet
 		return nil, nil
 	}
 	var rows []types.Row
-	if sctx.GetSessionVars().EnableChunk && rs.SupportChunk() {
+	if sctx.GetSessionVars().EnableChunk {
```

Comment:
We should remove EnableChunk too.

Reply:
will be done in next PRs.

Reply:
EnableChunk will be removed after removing Next.